nupic/test/test_framework_helpers.py

```diff
@@ -64,3 +66,50 @@ def testSwarm(self):
       pass
   """
   return getattr(pytest.mark, tag)
+
```

Comment:
This test is same that this one: https://github.com/david-ragazzi/nupic/commit/d78651f689f343ec817611bc32e51274f57dfb88